storage/integration_test.go

```diff
@@ -759,8 +759,16 @@ func TestIntegration_ObjectReadGRPC(t *testing.T) {
 
 	content := []byte("Hello, world this is a grpc request")
 
-	var name string
-	if name, err = writeTestData(ctx, grpcBucket, content); err != nil {
+	// Create an HTTP client to upload test data.
+	hc, err := NewClient(ctx)
```

Comment:
Use [`testConfig`](https://github.com/googleapis/google-cloud-go/blob/e36b29a3d43bce5c1c044f7daf6e1db00b0a49e0/storage/integration_test.go#L192) for client creation (actually, we should make a version of this method that creates a gRPC client as well).

Reply:
Done! I haven't done the grpc-flavored `testConfig`, but I put the HTTP `testConfig` call ahead of grpc client creation so that it'll skip if necessary. Can we do the grpc-flavored `testconfig` in a follow on PR?

Reply:
Sure, that's fine!

Reply:
I added a `testConfigGRPC` that creates both HTTP and gRPC clients.